test/e2e/sagemaker/__init__.py

```diff
@@ -23,3 +26,23 @@
 
 # PyTest marker for the current service
 service_marker = pytest.mark.service(arg=SERVICE_NAME)
+
+def make_reference(resource_prural, resource_name, spec_file, replacements, namespace="default"):
```

Comment:
nit: I don't think this name accurately enough describes its function. Perhaps `load_resource_and_reference`? I do think it is a bit odd that the method also returns a tuple of two unrelated objects - the reference is not at all related to the spec.
also nit: These could be used by all services, so we could put this in the `common` directory instead.
also also nit: Please add docstrings to these helper methods

Reply:
done, will do a minor PR to add doc strings